todo-mvc/src/widgets/createFocusableTextInput.ts

```diff
@@ -20,7 +20,9 @@ const afterUpdateFunctions = new WeakMap<FocusableTextInput, {(element: HTMLInpu
 function afterUpdate(instance: FocusableTextInput, element: HTMLInputElement) {
 	const focused = instance.state.focused;
 	if (focused) {
-		setTimeout(() => element.focus(), 0);
+		if (focused) {
+			setTimeout(() => element.focus(), 0);
```

Comment:
Oh, hah, probably missed the double `if` when rebasing. For a while it didn't need the setTimeout, but now it seems to again. I'm not sure if it's a maquette change or something we did.